packages/toolkit/scripts/build/generateConfigurationAttributes.ts

```diff
@@ -49,6 +38,31 @@ async function generateConfigurationAttributes(): Promise<void> {
         if (debuggerConfig === undefined) {
             continue
         }
+
+        if (!fs.existsSync(path.dirname(debuggerConfig.outputFile))) {
+            throw new Error(`Output folder of ${debuggerConfig.outputFile} does not exist.`)
+        }
+
+        const header = `
+/*!
+* Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
```

Comment:
Should we make this copyright blob a global constant? We'll probably create other generated files in the future. Maybe in a `scripts/constants.ts`

Reply:
good idea 👍

Reply:
will create in a follow-up